.github/workflows/push-xc-admin-frontend-image.yml

```diff
@@ -1,6 +1,5 @@
 name: Build and Push Cross Chain Admin Frontend
 on:
-  pull_request:
```

Comment:
this is the only docker build / push that happens on PRs right now. It's not necessary to run on PRs though, because the xc_admin_frontend code is built and tested in the typescript ci workflow already.

Reply:
Typescript ci test is not enough for this. This workflow was initially on main and got broken couple of times. It is because one should update the dockerfiles when a dependency added to the project.

Reply:
seems like it was added here : https://github.com/pyth-network/pyth-crosschain/commit/92aa157e856bfa500130f33d5ac1b429b9c4b2f2

Reply:
I see. I think it's ok if it fails from time to time. We don't change the xc_* very often anymore.